src/components/RentDialog/index.tsx

```diff
@@ -30,15 +38,20 @@ const RentDialog = ({ isOpen, handleClose, costs }: RentDialogProps) => {
             startDate: moment(),
             endDate: moment().add(1, 'days'),
           }}
-          onSubmit={(values, { setSubmitting }) => {
-            console.log(values);
-            setSubmitting(true);
-            setTimeout(() => {
-              setSubmitting(false);
-            }, 1500);
+          onSubmit={async (values) => {
+            try {
+              setWasCalled(true);
+              await AddReservation({
+                variables: {
+                  endAt: moment(values.endDate).format('YYYY-MM-DD') + 'T00:00:00.000+00:00',
+                  startAt: moment(values.startDate).format('YYYY-MM-DD') + 'T00:00:00.000+00:00',
+                  annoucementId: id,
+                },
```

Comment:
If possible, I suggest moving it above to better readability. This will prevent someone from having to search that specific fragment, somewhere in the code.

Reply:
ok, i can move that to separate file

Reply:
The separate file is unnecessary. Just create at the beginning of component object with that properties 😄

Reply:
I did as you wanted 😀